File: design-pattern-jvm/app/src/main/java/design/pattern/java/patterns/structural/bridge/ServiceBridge.java
package design.pattern.java.patterns.structural.bridge;

public class ServiceBridge {
    private final SomeLibrary someLibrary;
    public ServiceBridge(SomeLibrary someLibrary) {
        this.someLibrary = someLibrary;
    }
    public String concreteFeature() {
        return someLibrary.writeString()
            .concat(someLibrary.writeOtherString())
            .concat(someLibrary.integerGenerator().toString());
    }
}

File: design-pattern-jvm/app/src/main/kotlin/design/pattern/kotlin/patterns/structural/bridge/ServiceBridge.kt
package design.pattern.kotlin.patterns.structural.bridge

class ServiceBridge(private val someLibrary: SomeLibrary) {
    fun concreteFeature(): String {
        return someLibrary.writeString() + someLibrary.writeOtherString() +
                someLibrary.integerGenerator().toString()
    }
}